src/containers/Ledgers/LedgerMetrics.tsx

```diff
@@ -90,7 +104,7 @@ export const LedgerMetrics = ({
     .map((key) => {
       let content: any = null
 
-      let className = 'label'
+      const className = 'label'
```

Comment:
You can remove this variable and use the value inline.

Reply:
Great call! Fixed in [446d43c](https://github.com/ripple/explorer/pull/1175/commits/446d43c78a8c75f036e10dfc526574eb744b8b1d)